www/rustup.css

```diff
@@ -31,6 +31,15 @@
     src: local('Work Sans Medium'), url("fonts/WorkSans-Medium.ttf") format('truetype');
 }
 
+@font-face {
+    font-family: 'Alfa Slab One';
```

Comment:
Can you squash this change into the previous commit? Otherwise the previous commit is left incomplete, which seems unfortunate.

Reply:
Squashed. Thank you!